deepmd/utils/argcheck.py

```diff
@@ -754,10 +779,19 @@ def gen_json(**kwargs):
         model_args(),
         learning_rate_args(),
         loss_args(),
+        loss_dict_args(),
         training_args(),
         nvnmd_args(),
     ), cls=ArgumentEncoder)
 
+def gen_args(**kwargs):
+    return [nvnmd_args(),
+            training_args(),
+            loss_dict_args(),
+            loss_args(),
+            learning_rate_args(),
+            model_args()]  # show in reverse order
```

Comment:
I may fix it in dargs and a separated PR if it is a bug..

Reply:
I fix the bug in https://github.com/deepmodeling/dargs/pull/22. May I release a new version of dargs?

Reply:
I release dargs v0.3.4. It's better to bump the dargs version for docs

https://github.com/deepmodeling/deepmd-kit/blob/060311eb972f25ee62ccd08efc4c880d2cc6afc2/setup.py#L83

Reply:
Done in this PR.